test/functional/wallet_migration.py

```diff
@@ -781,30 +774,30 @@ def send_to_script(script, amount):
         assert_equal(next((it['desc'] for it in wallet_wo.listdescriptors()['descriptors'] if it['desc'] == desc_invalid), None), None)
 
         # Just in case, also verify wallet restart
-        self.nodes[0].unloadwallet(info_migration["watchonly_name"])
-        self.nodes[0].loadwallet(info_migration["watchonly_name"])
+        self.master_node.unloadwallet(info_migration["watchonly_name"])
+        self.master_node.loadwallet(info_migration["watchonly_name"])
         assert_equal(wallet_wo.getbalances()['mine']['trusted'], 5)
 
     def test_conflict_txs(self):
         self.log.info("Test migration when wallet contains conflicting transactions")
-        def_wallet = self.nodes[0].get_wallet_rpc(self.default_wallet_name)
+        def_wallet = self.master_node.get_wallet_rpc(self.default_wallet_name)
 
         wallet = self.create_legacy_wallet("conflicts")
         def_wallet.sendtoaddress(wallet.getnewaddress(), 10)
```

Comment:
I have noticed this pattern in most of these tests that I found a little difficult to read. Slightly more verbose naming would make it easier to parse like below. I am not suggesting this change to be done in this PR because this pattern existed earlier as well but can be updated in the future if others find this helpful as well.   

```diff
- def_wallet.sendtoaddress(wallet.getnewaddress(), 10)
+ def_descriptor_wallet.sendtoaddress(legacy_wallet.getnewaddress(), 10)
```